### PR TITLE
Update autoscaler version

### PR DESCRIPTION
**Pull request by bhack**

### CODE REVIEW

File: stable/cluster-autoscaler/values.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ cloudProvider: aws
 
 image:
   repository: gcr.io/google_containers/cluster-autoscaler
-  tag: v0.6.0
+  tag: v0.6.1
```
Comment (timstoop) [inline stable/cluster-autoscaler/values.yaml]: Why not to 0.6.2?

**Review by timstoop — CHANGES_REQUESTED** [review]
As charts are immutable, you need to bump the version in Charts.yaml as well.

File: stable/cluster-autoscaler/values.yaml (hunk 1 above)
Comment (bhack) [reply stable/cluster-autoscaler/values.yaml]: I don't know the 0.6.2 was released. The is also an 1.x series. Also I think that we need to maintain the yaml quite in sync with upstream. We really need to have real maintainers and MIA/ORPHANED for stable charts.

**Comment by unguiculus** [comment]
Marking stale. Pleae update within one week.

**Comment by bhack** [comment]
[1.1.0 released](https://github.com/kubernetes/autoscaler/releases/tag/cluster-autoscaler-1.1.0)

File: stable/cluster-autoscaler/Chart.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: Scales worker nodes within autoscaling groups.
 icon: https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png
 name: cluster-autoscaler
-version: 0.2.1
+version: 1.1.0
```
Comment (unguiculus) [inline stable/cluster-autoscaler/Chart.yaml]: That's a huge version bump. Please add `appVersion`, which is missing anyways.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/cluster-autoscaler/values.yaml [hunk 3]
```diff
@@ -11,7 +11,7 @@ cloudProvider: aws
 
 image:
   repository: gcr.io/google_containers/cluster-autoscaler
-  tag: v0.6.0
+  tag: v1.1.0
```
Comment (timstoop) [inline stable/cluster-autoscaler/values.yaml:14]: v1.1 is for Kubernetes 1.9, according to the docs. The v1.0 is for k8s 1.8 and v0.6.x (latest being v0.6.2, last time I checked) is for k8s 1.7. From what I gather, I don't think we want to upgrade to a version for a k8s that's still in beta?

**Comment by bhack** [comment]
It was not always clear what is really what. See  https://github.com/kubernetes/autoscaler/issues/346#issuecomment-351348942

**Comment by bhack** [comment]
http://blog.kubernetes.io/2017/12/kubernetes-19-workloads-expanded-ecosystem.html

**Comment by viglesiasce** [comment]
Hey @bhack @timstoop any word on whether this can work with 1.8 and 1.9?

**Comment by afirth** [comment]
Looks like this is already updated in master by https://github.com/kubernetes/charts/pull/3248
Suggest to close, the readme is also out of date

Re @timstoop 's comment, 1.1.0 is used by the kops addon for K8S 1.8. 1.1.1 dropped earlier this month

**Pull request closed without merging**